src/Compilers/VisualBasic/Test/Emit/CodeGen/CodeGenTuples.vb

```diff
@@ -1285,6 +1285,27 @@ Sum: 10, Count: 4")
         End Sub
 
         <Fact>
+        <WorkItem(18762, "https://github.com/dotnet/roslyn/issues/18762")>
+        Public Sub UnnamedTempShouldNotCrashPbdEncoding()
+
+            Dim verifier = CompileAndVerify(
+<compilation>
+    <file name="a.vb">
+Imports System
+Imports System.Threading.Tasks
+
+Module Module1
+    Private Async Function DoAllWorkAsync() As Task(Of (FistValue As String, SecondValue As String))
+        Return (Nothing, Nothing)
+    End Function
+End Module
+    </file>
+</compilation>,
+additionalRefs:={ValueTupleRef, SystemRuntimeFacadeRef}, useLatestFramework:=True, options:=TestOptions.DebugDll)
+
+        End Sub
+
+        <Fact>
```

Comment:
@AlekseyTs Using an empty string for SynthesizedLocal.Name is not trivial.
In particular, I found specific logic for using a `null` name for the function value locals in state machines:

```VB
      Private Function GetLocalDebugName(local As LocalSymbol, <Out> ByRef localId As LocalDebugId) As String
          ...
            ' We include function value locals in async and iterator methods so that appropriate
            ' errors can be reported when users attempt to refer to them.  However, there's no
            ' reason to actually emit them into the resulting MoveNext method, because they will
            ' never be accessed.  Unfortunately, for implementation-specific reasons, dropping them
            ' would be non-trivial.  Instead, we drop their names so that they do not appear while
            ' debugging (esp in the Locals window).
            If local.DeclarationKind = LocalDeclarationKind.FunctionValue AndAlso
                TypeOf _method Is SynthesizedStateMachineMethod Then
                Return Nothing
            End If
```
I'll stick with current approach if that's ok.